Update ctl_automate_server.rst

jeremiahsnapp opened the pull request:
Files in `~/.chef/client.d` get copied to the runner's `/etc/chef/client.d` directory

chef-delivery commented:
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

jamescott commented:
@delivery approve

chef-delivery commented:
Change: 0513d09e-bb43-41b3-ad5d-450a59564b13 approved by: @jamescott

jeremiahsnapp committed:
Update ctl_automate_server.rst

Files in `~/.chef/client.d` get copied to the runner's `/etc/chef/client.d` directory

jeremiahsnapp committed:
Update ctl_automate_server.rst

Commit:
Copyedit and fixed a link

Signed-off-by: David Wrede <dwrede@chef.io>

davidwrede commented:
![tenor-221939476](https://cloud.githubusercontent.com/assets/1634752/26706094/7cc9b8be-46ef-11e7-91b9-3ef3fdd9fd00.gif)
